doc-content/enterprise-only/installation/nfs-server-configure_proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='nfs-server-configure_{context}']
+= Configuring an NFS server
+
+You must deploy and configure an NFS server to provide the file system necessary for a {CENTRAL} clustered environment.
```

Comment:
Question - does this NFS procedure need to be added as a prerequisite to any other procedure in this guide?

Reply:
@csherrar well not just the NFS procedure but also the datagrid one are prerequisites for the BC deployment, you want me to make that exlpicit?

Reply:
@mramendi yes, I think we should. What do you think?

Reply:
@csherrar wait, it's already there as a prereq.